contracts/LifMarketValidationMechanism.sol

```diff
@@ -62,6 +62,9 @@ contract LifMarketValidationMechanism is Ownable {
   event Pause();
   event Unpause(uint256 pausedSeconds);
 
+  event ClaimedEth(uint256 claimedWei);
```

Comment:
it shouldnt be `claimedWei`? Its in wei units :)

Reply:
I wanted to keep it consistent with the corresponding method name `claimEth` which also deals with wei amounts but it has eth in its name :)

Should we change the method name too? Maybe in a subsequent PR?

Reply:
Yes I think we should, it claims it in Wei unit. So the next PR can be change Eth to Wei in naming.

Reply:
Cool, I'll change both in a further PR, thanks!